client/src/components/Header/Header.js

```diff
@@ -1,14 +1,24 @@
-import React from "react";
+import React, { useState } from "react";
 import styled from "@emotion/styled";
 import logoSrc from "../../assets/logo.svg";
 import MenuIconSrc from "../../assets/menu.svg";
+// import { BrowserRouter as Router, Link } from "react-router-dom";
+import Menu from "../menu/Menu";
+
+const Header = (props) => {
```

Comment:
If you don't use props, you can savely remove them
```suggestion
const Header = () => {
```